app/presenters/sufia/file_set_presenter.rb

```diff
@@ -4,6 +4,14 @@ class FileSetPresenter < ::CurationConcerns::FileSetPresenter
 
     delegate :depositor, :tag, :date_created, :date_modified, to: :solr_document
 
+    def collector
```

Comment:
Push the discussion of collector to a separate ticket regarding roles and role management?